elements/pf-dropdown/pf-dropdown-group.ts

```diff
@@ -15,17 +16,60 @@ export class PfDropdownGroup extends LitElement {
 
   static override readonly shadowRootOptions: ShadowRootInit = { ...LitElement.shadowRootOptions, delegatesFocus: true };
 
+  /**
+   * Disable the dropdown
+   */
+  @property({ type: Boolean, reflect: true }) disabled = false;
+
   /**
    * The label for the group of dropdown items.
    */
   @property({ reflect: true }) label?: string;
 
+
+  #internals = InternalsController.of(this, { });
```

Comment:
```suggestion
  #internals = InternalsController.of(this);
```
If we're not availing ourselves of the options, we [shouldn't need to supply](https://github.com/patternfly/patternfly-elements/blob/1b8f5aebfe280570df1a863f11afe1d4c61c10da/core/pfe-core/controllers/internals-controller.ts#L66) an empty object.